### PR TITLE
feat(config): import device configs from Z-Wave Alliance (Part 3: first substantive push of devices)

### PR DESCRIPTION
**Pull request by blhoward2**
First substantive push of new devices or changes to parameters. #1349 

Note: I'm adding the new double tap support to a large number of GE and Honeywell switches (both made by Jasco), based on the group names in the zwa files. Group 3 = "Double Tap" in the files confirming support.

### CODE REVIEW

**Comment by blhoward2** [comment]
@zwave-js-bot fix lint

**Comment by blhoward2** [comment]
Note, the one warning about duplicate values for the Sigma Powerstrip...I have no idea what to put for that device. I can't locate it. I actually think the entire file should probably be deleted? The manufacturerID, ProductType, and Product Id are all 0x0000. How would that even work?

**Comment by AlCalzone** [comment]
AFAIK that is the very first Z-Wave device by Sigma Designs before they scrapped the hardware and focused on the protocol.
Deleting it probably won't hurt that much.

**Review by robertsLando — DISMISSED** [review]
I have give all files a quick look, seems good! I would keep the `\n` in metadata and remove that in values description

**Comment by AlCalzone** [comment]
On a side note, I need to figure out why the bot comments after each failed commit. That should not happen this frequently.

File: packages/config/config/devices/0x0039/39339_zw3107.json [hunk 1]
```diff
@@ -1,35 +1,52 @@
-// Honeywell 39339 (ZW3107)
+// Honeywell 39339 / 39446 / ZW3107
 // Z-Wave Plug-in Smart Dimmer
 {
 	"manufacturer": "Honeywell",
 	"manufacturerId": "0x0039",
-	"label": "39339 (ZW3107)",
+	"label": "39339 / 39446 / ZW3107",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0039/39339_zw3107.json:6]: I'm not sure if we should group devices with a different label just becausue they have the same parameters.
That could lead to confusion.

File: packages/config/config/devices/0x0039/39348-zw4008.json [hunk 2]
```diff
@@ -1,12 +1,15 @@
+// Honeywell 39348/ZW4008
+// 39348 / ZW4008 In-Wall Smart Switch
 {
 	"manufacturer": "Honeywell",
 	"manufacturerId": "0x0039",
-	"label": "39348-ZW4008",
+	"label": "39348/ZW4008",
 	"description": "39348 / ZW4008 In-Wall Smart Switch",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0039/39348-zw4008.json]:
```suggestion
	"label": "39348 / ZW4008",
	"description": "In-Wall Smart Switch",
```

File: packages/config/config/devices/0x0039/39348-zw4008.json [hunk 3]
```diff
@@ -73,14 +76,22 @@
 			"allowManualEntry": false,
 			"options": [
 				{
-					"label": "Any button",
+					"label": "Press any button on the device",
 					"value": 0
 				},
 				{
-					"label": "2x ON button, 2x OFF button",
+					"label": "Press 2 times ON button and then 2 times OFF button, LED will flash 5 times if done successful",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0039/39348-zw4008.json]: Thats a bit long for a drop down or similar
```suggestion
					"label": "Press ON button 2x, then OFF button 2x",
```

File: packages/config/config/devices/0x0039/39351_jasco_zw3005.json [hunk 4]
```diff
@@ -24,12 +40,13 @@
...
 			"maxValue": 2,
 			"defaultValue": 0,
+			"unsigned": true,
 			"readOnly": false,
 			"writeOnly": false,
 			"allowManualEntry": false,
 			"options": [
 				{
-					"label": "LED ON when load is OFF",
+					"label": "LED ON when load is OFF (Default)",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0039/39351_jasco_zw3005.json]: I don't like mentioning in the label that an option is the default

File: packages/config/config/devices/0x0039/39351_jasco_zw3010.json [hunk 5]
```diff
@@ -1,44 +1,65 @@
...
+			"maxValue": 3,
 			"defaultValue": 0,
+			"unsigned": true,
 			"readOnly": false,
 			"writeOnly": false,
 			"allowManualEntry": false,
 			"options": [
 				{
-					"label": "LED ON when load is OFF",
+					"label": "LED ON when load is OFF (Default)",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0039/39351_jasco_zw3010.json]: dito

File: packages/config/config/devices/0x0039/39354.json [hunk 6]
```diff
@@ -1,22 +1,73 @@
...
+			"minValue": 0,
+			"maxValue": 3,
+			"defaultValue": 0,
+			"unsigned": true,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "LED ON when load is OFF (Default)",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0039/39354.json]: dito

File: packages/config/config/devices/0x0039/39357_zw3004.json [hunk 7]
```diff
@@ -1,97 +1,169 @@
...
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Default",
+					"value": 0
+				},
+				{
+					"label": "Invert",
+					"value": 1
+				}
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0039/39357_zw3004.json]:
```suggestion
				{
					"label": "Not inverted",
					"value": 0
				},
				{
					"label": "Inverted",
					"value": 1
				}
```

File: packages/config/config/devices/0x0039/39357_zw3011.json [hunk 8]
```diff
@@ -0,0 +1,180 @@
...
+			"unsigned": true,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": true,
+			"options": [
+				{
+					"label": "1=1%, 99=99%",
+					"value": 99
+				}
+			]
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0039/39357_zw3011.json]:
```suggestion
```

File: packages/config/config/devices/0x0084/mimolite.json [hunk 9]
```diff
@@ -64,7 +65,7 @@
 			"label": "Clear_Pulse_Meter_Counts",
 			"valueSize": 1,
 			"minValue": 0,
-			"maxValue": 255,
+			"maxValue": 0,
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0084/mimolite.json:68]: Can you really only set 0 here? Then I'd probably also set `allowManualEntry: false`

File: packages/config/config/devices/0x0086/zw090.json [hunk 10]
```diff
@@ -33,90 +36,96 @@
...
+			"label": "Reset the configuration parameter to default factory setting",
+			"description": "Resets all configuration parameters to default setting",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 0,
+			"defaultValue": 0,
+			"unsigned": true,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": true
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0086/zw090.json:125]: The Aeotec devices usually have some weird value (not 0) that needs to be put here.

File: packages/config/config/devices/0x0109/zd2201.json [hunk 11]
```diff
@@ -24,17 +25,19 @@
...
 			"maxValue": 1,
 			"defaultValue": 0,
+			"unsigned": true,
 			"readOnly": false,
 			"writeOnly": false,
 			"allowManualEntry": false,
 			"options": [
 				{
-					"label": "Celsius (default)",
+					"label": "Celsius (Default)",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0109/zd2201.json]:
```suggestion
					"label": "Celsius",
```

File: packages/config/config/devices/0x0109/zd2201.json [hunk 12]
```diff
@@ -44,39 +47,48 @@
 			]
 		},
 		"2": {
-			"label": "Temperature Reporting Threshold",
-			"description": "Set up from 0.1C to 5C",
+			"label": "Auto Temperature Report Interval",
+			"description": "Auto Temperature Report Interval from 0.1°C to 5°C",
 			"valueSize": 1,
-			"minValue": 1,
+			"unit": "°C",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0109/zd2201.json]:
```suggestion
			"unit": "0.1 °C",
```

File: packages/config/config/devices/0x010f/fgpb101.json [hunk 13]
```diff
@@ -53,179 +91,460 @@
...
+				},
+				{
+					"label": "Key Pressed 4 times",
+					"value": 8
+				},
+				{
+					"label": "Key Pressed 5 times",
+					"value": 16
+				}
+			]
```
Comment (AlCalzone) [inline packages/config/config/devices/0x010f/fgpb101.json]: That's a bitmask. We need to convert this to partial params

File: packages/config/config/devices/0x010f/fgpb101.json [hunk 14]
```diff
@@ -53,179 +91,460 @@
...
+				},
+				{
+					"label": "3rd Group Sent as Secure",
+					"value": 2
+				},
+				{
+					"label": "4th Group Sent as Secure",
+					"value": 4
+				}
+			]
```
Comment (AlCalzone) [inline packages/config/config/devices/0x010f/fgpb101.json]: This is also a bitmask

File: packages/config/config/devices/0x010f/fgpb101.json [hunk 15]
```diff
@@ -53,179 +91,460 @@
...
 		},
 		"11": {
-			"label": "Key Pressed 1 time – value ON sent to 2nd group",
-			"description": "Value of SWITCH ON command sent to devices in 2nd association group",
+			"label": "Key Pressed 1 time – value of SWITCH ON command sent to 2nd association group",
+			"description": "1-99 Set Level; 255 Last State",
 			"valueSize": 2,
-			"minValue": 1,
+			"minValue": 99,
```
Comment (AlCalzone) [inline packages/config/config/devices/0x010f/fgpb101.json]: The description is correct, minValue is not.

File: packages/config/config/devices/0x010f/fgpb101.json [hunk 16]
```diff
@@ -53,179 +91,460 @@
...
+			]
 		},
 		"13": {
-			"label": "Key Pressed 2 times – value ON sent to 2nd group",
-			"description": "SWITCH ON command in 2nd association group after double click",
+			"label": "Key Pressed 2 times – value of SWITCH ON command sent to 2nd association group",
+			"description": "1-99 Set Level; 255 Last State",
 			"valueSize": 2,
-			"minValue": 1,
+			"minValue": 99,
```
Comment (AlCalzone) [inline packages/config/config/devices/0x010f/fgpb101.json]: dito

File: packages/config/config/devices/0x010f/fgpb101.json [hunk 17]
```diff
@@ -53,179 +91,460 @@
...
 		},
 		"15": {
-			"label": "Key Pressed 3 times – value ON to 2nd group",
-			"description": "SWITCH ON sent in 2nd association group after 3 presses",
+			"label": "Key Pressed 3 times – value of SWITCH ON command sent to 2nd association group",
+			"description": "1-99 Set Level; 255 Last State",
 			"valueSize": 2,
-			"minValue": 1,
+			"minValue": 99,
 			"maxValue": 255,
```
Comment (AlCalzone) [inline packages/config/config/devices/0x010f/fgpb101.json]: dito

File: packages/config/config/devices/0x010f/fgpb101.json [hunk 18]
```diff
@@ -53,179 +91,460 @@
...
 		},
 		"21": {
-			"label": "Key Pressed 1 time – value ON sent to 3rd group",
-			"description": "Value of SWITCH ON command sent in 3rd association group after a single click.",
+			"label": "Key Pressed 1 time – value of SWITCH ON command sent to 3rd association group",
+			"description": "1-99 Set Level; 255 Last State",
 			"valueSize": 2,
-			"minValue": 1,
+			"minValue": 99,
 			"maxValue": 255,
```
Comment (AlCalzone) [inline packages/config/config/devices/0x010f/fgpb101.json]: dito

File: packages/config/config/devices/0x010f/fgpb101.json [hunk 19]
```diff
@@ -53,179 +91,460 @@
...
 		},
 		"23": {
-			"label": "Key Pressed 2 times – value ON sent to 3rd group",
-			"description": "Value of SWITCH ON command sent in 3rd association group after a double click.",
+			"label": "Key Pressed 2 times – value of SWITCH ON command sent to 3rd association group",
+			"description": "1-99 Set Level; 255 Last State",
 			"valueSize": 2,
-			"minValue": 1,
+			"minValue": 99,
 			"maxValue": 255,
```
Comment (AlCalzone) [inline packages/config/config/devices/0x010f/fgpb101.json]: dito

File: packages/config/config/devices/0x010f/fgpb101.json [hunk 20]
```diff
@@ -53,179 +91,460 @@
...
 		},
 		"25": {
-			"label": "Key Pressed 3 times – value ON sent to 3rd group",
-			"description": "Value SWITCH ON command sent in 3rd association group after a triple click.",
+			"label": "Key Pressed 3 times – value of SWITCH ON command sent to 3rd association group",
+			"description": "1-99 Set Level; 255 Last State",
 			"valueSize": 2,
-			"minValue": 1,
+			"minValue": 99,
 			"maxValue": 255,
```
Comment (AlCalzone) [inline packages/config/config/devices/0x010f/fgpb101.json]: dito

File: packages/config/config/devices/0x010f/fgpb101.json [hunk 21]
```diff
@@ -53,179 +91,460 @@
...
+				},
+				{
+					"label": "Key Held Down",
+					"value": 32
+				},
+				{
+					"label": "Key Released",
+					"value": 64
+				}
+			]
```
Comment (AlCalzone) [inline packages/config/config/devices/0x010f/fgpb101.json]: Bitmask :)

File: packages/config/config/devices/0x0129/yale_smart_door_lock_z-wave_module.json [hunk 22]
```diff
@@ -21,5 +27,84 @@
...
+			"options": [
+				{
+					"label": "Auto relock Off",
+					"value": 0
+				},
+				{
+					"label": "Aut Relock On",
+					"value": 255
+				}
+			]
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0129/yale_smart_door_lock_z-wave_module.json]:
```suggestion
			"options": [
				{
					"label": "Off",
					"value": 0
				},
				{
					"label": "On",
					"value": 255
				}
			]
```

File: packages/config/config/devices/0x0129/yale_smart_door_lock_z-wave_module.json [hunk 23]
```diff
@@ -21,5 +27,84 @@
...
+			"allowManualEntry": true
+		},
+		"6": {
+			"label": "Remote Relock Time",
+			"description": "Amount of time (secs)",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 10,
+			"defaultValue": 10,
+			"unsigned": true,
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0129/yale_smart_door_lock_z-wave_module.json:100]:
```suggestion
			"unit": "seconds",
			"unsigned": true,
```

File: packages/config/config/devices/0x0131/ne_nas_ab02z.json [hunk 24]
```diff
@@ -0,0 +1,160 @@
...
+					"label": "High",
+					"value": 3
+				}
+			]
+		},
+		"2": {
+			"label": "Configure alarm duration time",
+			"description": "30 sec, 1 min, 5 min",
+			"valueSize": 1,
+			"minValue": 0,
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0131/ne_nas_ab02z.json:63]: 1?

**Review by AlCalzone — CHANGES_REQUESTED** [review]
In general: Lets keep "(default)" out of the descriptions and labels.

File: packages/config/config/devices/0x0039/39339_zw3107.json (hunk 1 above)
Comment (AlCalzone) [reply packages/config/config/devices/0x0039/39339_zw3107.json:6]: Seems to be the only occurence of this though.

Comment (blhoward2) [reply packages/config/config/devices/0x0039/39339_zw3107.json:6]: These share a product type/id so no option but to group them.

File: packages/config/config/devices/0x0131/ne_nas_ab02z.json (hunk 24 above)
Comment (blhoward2) [reply packages/config/config/devices/0x0131/ne_nas_ab02z.json:63]: @AlCalzone That’s the default in the file and those three options are provided like that. I’m not sure that it’s unusual you’d want a siren to go off for a while if activated. Do you prefer a different default?

File: packages/config/config/devices/0x0084/mimolite.json (hunk 9 above)
Comment (blhoward2) [reply packages/config/config/devices/0x0084/mimolite.json:68]: Yes, it seems to clear the counter when you send zero (as far as I can tell)

**Comment by blhoward2** [comment]
@AlCalzone I did a system-wide removal of (Default) from labels so some new files were brought in. I'd copied what we had elsewhere. I think I still have your comment about the Aeotec device and the bitmask open.

**Comment by blhoward2** [comment]
@AlCalzone After thinking more about this, I actually like to know what the default was when troubleshooting and changing a bunch of things. For some things like dimming steps it is nice to know how far from the default I’ve strayed. 

What if we kept the label but made it consistent? I could parse every file and add however to each default option value. 

I don’t care that much as I know how to figure out the default but others may care too.

**Comment by blhoward2** [comment]
@zwave-js-bot fix lint

File: packages/config/config/devices/0x010f/fgpb101.json (hunk 21 above)
Comment (blhoward2) [reply packages/config/config/devices/0x010f/fgpb101.json]: @AlCalzone I'm removing this part for now, unless you can tell me how it should look. I haven't delved into the bitmask ones yet.

File: packages/config/config/devices/0x010f/fgpb101.json (hunk 13 above)
Comment (blhoward2) [reply packages/config/config/devices/0x010f/fgpb101.json]: I went back to manual entry for now, unless you can tell me how it should look. I haven't had time to look into it yet.

File: packages/config/config/devices/0x0086/zw090.json (hunk 10 above)
Comment (blhoward2) [reply packages/config/config/devices/0x0086/zw090.json:125]: Some do, but not all. The zw089 also has only 0/1. From the manual.

Screen Shot 2021-01-26 at 12 08 39 AM

**Comment by blhoward2** [comment]
@AlCalzone Two comments for you above. Otherwise I think these are good to go.

**Review by AlCalzone — CHANGES_REQUESTED** [review]
I agree with all the changes, except fgpb101.json
The Bitmask is missing and the max value is incorrect (if all bits are set, the max is 127)

**Comment by blhoward2** [comment]
I removed the bitmask for now as I don’t have time to look at it today and planned to go back to add it when I start doing the bitmasks. Or I can try to change that file back to its original so it drops out altogether.

**Comment by AlCalzone** [comment]
Alright, I'll fix it, so you have an example for your next group of devices.

**Review by AlCalzone — APPROVED** [review]

**Pull request merged**